lib/srv/term.go

```diff
@@ -617,7 +613,7 @@ func (t *remoteTerminal) Wait() (*ExecResult, error) {
 	}, nil
 }
 
-func (t *remoteTerminal) WaitForChild() error {
+func (t *remoteTerminal) WaitForChild(_ context.Context) error {
```

Comment:
If you take the suggestion from above.
```suggestion
func (t *remoteTerminal) WaitForChild(context.Context) error {
```